README.md

```diff
@@ -19,7 +19,7 @@ Queue That is built primarily to queue XHR requests for reporting.
 
 - When more than one tab is open, only one active queue will process tasks.
 - If `options.process` calls back with an error, the tasks will be passed to `options.process`
-  again after a second. This backoff time doubles for each sequential error. The backoff timer is
+  again after a second. This backoff time doubles for each sequential error, up to `options.maxBackoff` (5 minutes by default). The backoff timer is
```

Comment:
Could we add this config option into the API documentation section below?

Reply:
@roberttod